runtime/src/asset.rs

```diff
@@ -419,6 +421,11 @@ decl_module! {
             for (typ, val) in &identifiers {
                 <Identifiers>::insert((ticker, typ.clone()), val.clone());
             }
+            // Add funding round name
+            match funding_round {
```

Comment:
Ahead of Clippy refactoring, this can be reduced to
```rust
if let Some(round) = funding_round {
    <FundingRound>::insert(ticker, round);
}
```